app/uploaders/profile_photo_uploader.rb

```diff
@@ -10,7 +10,7 @@ def store_dir
     "public/testuploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
   end
 
-  def default_url
-    ActionController::Base.helpers.asset_path("fallback/" + [version_name, "https://daily-buzz-profile-pics.s3.amazonaws.com/coffee-791045_1280.jpg"].compact.join('_'))
-  end
+  # def default_url
+  #   ActionController::Base.helpers.asset_path("fallback/" + [version_name, "https://daily-buzz-profile-pics.s3.amazonaws.com/coffee-791045_1280.jpg"].compact.join('_'))
+  # end
```

Comment:
if this is commented out for now, I would just add this code to your trello board and remove. I get why you have it, but we do want to keep our PRs clean